tests/test_plugins/test_plugins.py

```diff
@@ -0,0 +1,273 @@
+import shutil
+import textwrap
+from pathlib import Path
+
+import pytest
+
+from ..utils.commands import capture
+
+plugin_pyproject_template = textwrap.dedent(
+    """
+        [tool.poetry]
+        name = "{plugin_name}"
+        authors = ["ManimCE"]
+        version = "0.1.0"
+        description = ""
+
+        [tool.poetry.dependencies]
+        python = "^3.6"
+
+        [tool.poetry.plugins."manim.plugins"]
+        "{plugin_name}" = "{plugin_entrypoint}"
+
+        [build-system]
+        requires = ["poetry-core>=1.0.0"]
+        build-backend = "poetry.core.masonry.api"
+        """
+)
+
+plugin_init_template = textwrap.dedent(
+    """
+    from manim import *
+    class {class_name}(VMobject):
+        def __init__(self):
+            super().__init__()
+            dot1 = Dot(fill_color=GREEN).shift(LEFT)
+            dot2 = Dot(fill_color=BLUE)
+            dot3 = Dot(fill_color=RED).shift(RIGHT)
+            self.dotgrid = VGroup(dot1, dot2, dot3)
+            self.add(self.dotgrid)
+
+        def update_dot(self):
+            self.dotgrid.become(self.dotgrid.shift(UP))
+    def {function_name}():
+        return [{class_name}]
+    """
+)
+
+cfg_file_contents = textwrap.dedent(
+    """
+        [CLI]
+        plugins = test_plugin
+    """
+)
+
+
+def simple_scenes_path():
+    return str(Path(__file__).parent / "simple_scenes.py")
+
+
+def cfg_file_create(cfg_file_contents, path):
+    file_loc = (path / "manim.cfg").absolute()
```

Comment:
Personally not too big a fan of the `/` operator on paths, I like to make it more explicit like `Path(path, "manim.cfg")` or you could use the `.joinpath` method, but the way it currently is is still fine

Reply:
I like this way though...

Reply:
I mean yeah fair enough I'm just not too big a fan of it, I'm not at all staunchly against the `/` operator here. Feel perfectly free to not change anything unless maybe someone else wants to throw in an opinion I guess

Reply:
The rest of the library seems to use 
`Path(your_file_name) / "rest" / "of" / "path"`

So I would just go for consistency here and do whatever is done *mostly* elsewhere in the codebase. Although, in general we should just have one style for these sorts of things.